README.md

```diff
@@ -253,6 +253,21 @@ ECR registry:
 This is useful if you use `docker` to operate on registries that use different
 authentication credentials.
 
+[!NOTE]
```

Comment:
I think we need to also update the configuration example in L225 that currently states `credsStore` and `auth` tokens working together.

Reply:
Removed the previous example, we now only suggest multiple repository settings with `credHelpers`